Classes/Controller/Backend/ControlCenter/TemplaVoilaPlus8UpdateController.php

```diff
@@ -1095,7 +1106,8 @@ function convertDsToForOneTo(array $allDs, array $to, array &$copiedBackendLayou
 
         GeneralUtility::writeFile(
             $publicExtensionDirectory . $innerPathes['templateConfiguration'][$convertedDsConfig['scopeName']] . '/' . $yamlFileName,
-            \Symfony\Component\Yaml\Yaml::dump($templateConfiguration, 100, 4, \Symfony\Component\Yaml\Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK) // No inline style please
+            \Symfony\Component\Yaml\Yaml::dump($templateConfiguration, 100, 4, \Symfony\Component\Yaml\Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK)
+            // No inline style please
```

Comment:
This comment is a hint to the Yaml::DUMP_MULTI_LINE_LITERAL_BLOCK ... now it misses its context